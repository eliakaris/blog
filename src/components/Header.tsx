import * as React from 'react';

function Header() {
  return (
    <header className="masthead">
      <div className="overlay" />
        <div className="container">
          <div className="row">
            <div className="col-lg-8 col-md-10 mx-auto">
              <div className="site-heading">
                <h1>Elia's blog</h1>
                <span className="subheading">A blog about learnings in web technology</span>
              </div>
            </div>
          </div>
        </div>
    </header>
  );
}

export default Header;