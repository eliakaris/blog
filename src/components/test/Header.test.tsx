import * as React from 'react';
import Header from '../Header';
import * as ShallowRenderer from 'react-test-renderer/shallow';
import * as ReactSixteenAdapter from 'enzyme-adapter-react-16';
import { configure } from 'enzyme';

configure({ adapter: new ReactSixteenAdapter() });

describe('<Header /> tests', () => {
  let renderer: any;

  beforeAll(() => {
    renderer = ShallowRenderer.createRenderer();
  });

  it('renders a snapshot', () => {
    const component = renderer.render(<Header />);
    expect(component).toMatchSnapshot();
  });
});