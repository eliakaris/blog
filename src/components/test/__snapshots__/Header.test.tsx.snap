// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> tests renders a snapshot 1`] = `
<header
  className="masthead"
>
  <div
    className="overlay"
  />
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-8 col-md-10 mx-auto"
      >
        <div
          className="site-heading"
        >
          <h1>
            Elia's blog
          </h1>
          <span
            className="subheading"
          >
            A blog about learnings in web technology
          </span>
        </div>
      </div>
    </div>
  </div>
</header>
`;
